circuit_maintenance_parser/parsers/cogent.py

```diff
@@ -0,0 +1,80 @@
+"""Cogent parser."""
+import logging
+import re
+from typing import Dict
+
+from datetime import datetime
+from bs4.element import ResultSet  # type: ignore
+
+from circuit_maintenance_parser.errors import ParsingError
+from circuit_maintenance_parser.parser import Html, Impact, CircuitImpact, Status
+
+
+logger = logging.getLogger(__name__)
+
+# pylint: disable=too-many-branches
+
+
+class ParserCogent(Html):
+    """Notifications Parser for Cogent notifications."""
+
+    provider_type: str = "cogent"
+
+    # Default values for cogent notifications
+    _default_provider = "cogent"
+    _default_organizer = "support@cogentco.com"
+
+    def parse_html(self, soup, data_base):
+        """Execute parsing."""
+        data = data_base.copy()
+        try:
+            self.parse_br(soup.find_all("br"), data)
+            self.parse_title(soup.find_all("title"), data)
+            return [data]
+
+        except Exception as exc:
+            raise ParsingError from exc
+
+    def parse_br(self, br_results: ResultSet, data: Dict):
+        """Parse <br> tag."""
+        for br in br_results:
+            div_text = br.parent.text
```

Comment:
Feels weird to me that we're searching for all `<br>` just so we can find the very first `<br>` and then use it to find the `<div>` that is its parent and iterate over that. Are there any distinguishing characteristics (maybe `class="a3s aiL "`?) that we can use to find the div directly?